src/app/dataset-block/metadata-block/components/event-details/components/base-event/base-event.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { mockSetPollingSourceEvent } from "../../mock.events";
+import { BaseEventComponent } from "./base-event.component";
+
+describe("BaseEventComponent", () => {
```

Comment:
I think these tests are unnecessary. It's an abstract class

Reply:
Ok